<launch>
   <arg name="id" />
   <arg name="launch_user" default="user"/>
   <arg name="launch_pass" default="pass"/>
   <param name="launch_user" value="$(arg launch_user)" />
   <param name="launch_pass" value="$(arg launch_pass)" />

   <node name="netcam_stream_$(arg id)" pkg="netcam_stream" type="netcam_stream_node" output="screen">
    <param name="camera_id" value="$(arg id)" />
   </node>
   <node name="image_proc_$(arg id)" pkg="image_proc" type="image_proc" ns="netcam_stream_$(arg id)" output="screen" />
</launch>
